HRInvoiceApp/HRInvoiceApp/Assignment_Page.xaml

```diff
@@ -0,0 +1,33 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="HRInvoiceApp.Assignment_Page">
+  <ListView x:Name="MyListView"
+            ItemsSource="{Binding Items}"
+            ItemTapped="Handle_ItemTapped"
+            CachingStrategy="RecycleElement">
+            
+    <!--Built in Cells-->
+    <ListView.ItemTemplate>
+        <DataTemplate>
+            <TextCell Text="{Binding .}" />
```

Comment:
Hoort die "." daar? Is dat de path? Als ie er hoort, negeer 👍

Reply:
Deze listview word later vervangen voor een nieuwe, nu staat deze er nog i.v.m. testen

Reply:
Top, approved :)